<!--
	This file will execute nodes written below, config parameters in this file.
 -->

<launch>

    <node name="serial_port_hub" pkg = "main_sequence" type = "serial_port_hub.py" output="screen">
		<param name="serial_port" value="/dev/ttyTHS2" type="str" />
		<param name="baudrate" value="115200" type="int" />
	 </node>

    <node name="main_execute_module" pkg = "main_sequence" type = "navigation_action_server.py" output="screen">
		<!-- The maximum distance, on which strategies to approach will take. -->
		<param name="MaxDistance" value="100" type="int" />
	 </node>

	<node name="control_hub" pkg = "main_sequence" type = "control_mode_hub.py" output="screen"/>

</launch>
